static/js/util/editEducation.js

```diff
@@ -41,9 +41,9 @@ export function openNewEducationForm(level: string, index: number) {
 }
 
 export function deleteEducationEntry () {
-  const { saveProfile, profile, ui, deletionIndex } = this.props;
+  const { saveProfile, profile, ui } = this.props;
```

Comment:
it's not super important, but you could pull out `deletionIndex` here with destructuring:

```js
const { saveProfile, profile, ui: { deletionIndex } } = this.props;
```